### ISSUE TITLE
message handler passes message by reference

### ISSUE THREAD

**Issue by AD7six** [Flotype/now#102]
This may be a misunderstanding on my part but

https://github.com/Flotype/now/blob/master/lib/now.js#L370

```
client.on('message', function (message){
  var messageObj = message;
  if(nowUtil.hasProperty(messageObj, "type") && nowUtil.hasProperty(nowCore.messageHandlers, messageObj.type)) {
      nowCore.messageHandlers[messageObj.type](client, messageObj.data);
  }
});
```

the messagObj var is simply a reference to message. I guess messageObj is supposed to be a copy of message, or is a left over from previous code.

**Comment by ericz** [comment]
This is probably left over from previous code

**Comment by steveWang** [comment]
Properly resolved as of f0c9919d33be756018fc56159d3cc7a899770840.
